designs/administrator/future-openshift/cluster-setup/index.md

```diff
@@ -0,0 +1,45 @@
+---
+parent: Administrator
+version: 4.X
+---
+
+# Cluster Setup
+
+## First time use
+![](img/setup-hint.png)
+When the user first signs into OpenShift after installation they are presented with a hint block that directs them to complete the cluster setup steps. Clicking on the action button in the hint should open the notification drawer.
+
+
+## Behavior
+Cluster setup items appear in the "recommendations" section of the notification drawer. Once a user has clicked on the action of the recommendation or if the cluster detects that the desired state has been reached, the item disappears from the notification drawer.
+
+## Contents
+![](img/setup-open.png)
+**Note: the exact phrasing and action targets of these items is not final**
+- Configure an identity provider
+  - "You are logged in as a temporary administrative user. Update the cluster OAuth configuration to allow others to log in."
+  - The action button for this item should kick off the "add identity provider" form.
+- Configure alert receivers
+  - "Tell OpenShift where to send important alerts about your cluster."
+  - This action should link to alertmanager configuration page
+- Default Ingress Certificate
+  - Replace the default ingress certificate to enable external clients to securely connect to your apps
+  - [This action should link to docs for ingress certificates](https://docs.openshift.com/container-platform/4.5/security/certificates/replacing-default-ingress-certificate.html)
+- Delete kubeadmin user secret
```

Comment:
Is this something that should be shown after OAuth setup and admin account creation?